components/WorkSectionAsGallery.js

```diff
@@ -20,6 +21,11 @@ const WorkGalleryAssetBlock = ({ assetBlock }) => {
     true
   );
   const works = get(assetBlock, 'fields.works', []);
+  const doubleColumnRowAlignment = get(
+    assetBlock,
+    'fields.doubleColumnRowAlignment',
```

Comment:
Naming is hard. I think this is pretty good, but maybe even `Double Column Vertical Alignment`? Can we also add a description in Contentful? "If set to top, the shorter of two assets of different height will vertically align to the top on tablet/desktop."

Reply:
yes @mokaymokay vertical alignment sounds giving more sense about the direction of alignment. So I changed the variable name and added description!